test/StreamManager.test.js

```diff
@@ -164,8 +168,58 @@ describe("StreamManager", function () {
       this.amount
     ))
     .to.be.revertedWith('NotPayer');
-  })
+  })  
+
+  // Tests for `accumulation();`
+  // Amount is accumulated
+  it('Return accumulated amount;', async () => {
+    // Create the open stream
+    await this.streamManager.createOpenStream(
+      this.payee1.address,
+      this.mockUSDT.address,
+      this.amount,
+      this.terminationPeriod,
+      this.cliffPeriod
+    );
+
+    // Setting tinestamp
+    const currentTime = Math.floor(Date.now() / 1000)
+    await time.increase(2 * 24 * 3600); // + 2 days
+
+    // Calling the `accumulation();`
+    const accumulatedAmount = await this.streamManager.connect(this.payee1).accumulation()
+    // Calculating expected amount
+    const expectedAmount = Math.floor(this.cliffPeriod * this.amount / 30 / 24 / 3600)
```

Comment:
Not correctly calculated. During cliff period, 0 amount is accumulated. Check existing tests